sql-cli/sql_cli/connections.py

```diff
@@ -49,17 +52,19 @@ def validate_connections(connections: list[Connection], connection_id: str | Non
     for connection in connections:
         if not connection_id or connection_id and connection.conn_id == connection_id:
             os.environ[f"AIRFLOW_CONN_{connection.conn_id.upper()}"] = connection.get_uri()
-            status = (
-                "[bold green]PASSED[/bold green]" if _is_valid(connection) else "[bold red]FAILED[/bold red]"
-            )
+            success_status, message = _is_valid(connection)
+            status = "[bold green]PASSED[/bold green]" if success_status else "[bold red]FAILED[/bold red]"
             rprint(f"Validating connection {connection.conn_id:{CONNECTION_ID_OUTPUT_STRING_WIDTH}}", status)
+            formatted_message = (
+                f"[bold green]{message}[/bold green]" if success_status else f"[bold red]{message}[/bold red]"
+            )
```

Comment:
@tatiana it probably makes sense to always print that message, no? 🤔 Otherwise we are not giving the users any hints on what's wrong with it.

Reply:
Yes, it does, @feluelle ! We should keep the the SQL CLI standard output regardless of the verbose mode :)